setup.py

```diff
@@ -122,6 +122,7 @@ def get_extras_require() -> Dict[str, List[str]]:
             "redis",  # optuna/storages/redis.py.
             "scikit-learn>=0.24.2",
             # optuna/visualization/param_importances.py.
+            "distributed",  # optuna/integration/dask.py
```

Comment:
I think this is not necessary. With `pytest.mark.integration`, `DaskStorage` is tested by `tests-integration`.